source/extensions/request_id/uuid/config.h

```diff
@@ -11,27 +11,29 @@ namespace RequestId {
 
 // UUIDRequestIDExtension is the default implementation if no other extension is explicitly
 // configured.
-class UUIDRequestIDExtension : public Http::RequestIDExtension {
+class UUIDRequestIDExtension : public Envoy::Http::RequestIDExtension {
```

Comment:
Why are these namespaces needed? Is there some include pollution? Perhaps this will be fixed via the loading trick I mentioned?

Reply:
Yeah namespace pollution.

Reply:
Let me check the loading trick to see if it helps / improves the situation. Sorry about the extra noise here, but I felt that tucking this under extensions/http is enough benefit to warrant the verboseness.

Reply:
Passing on this per the above comment.

Reply:
Sorry why are we passing this? I think you can clean up the code quite a bit?

Reply:
I thought the namespace cleanup was optional. Sure I can handle it.

Reply:
Cleaned.